src/users/me.controller.spec.ts

```diff
@@ -62,6 +64,38 @@ describe('MeController', () => {
           telegram: user.telegram,
         });
       });
+
+      it('returns the user for jwt token', async () => {
+        const user = await usersService.create({
+          email: faker.internet.email(),
+          graffiti: uuid(),
+          countryCode: faker.address.countryCode('alpha-3'),
+        });
+
+        const token = jwt.sign({ sub: user.email, iat: Date.now() }, 'secret', {
+          expiresIn: '1d',
+        });
+
+        const { body } = await request(app.getHttpServer())
+          .get('/me')
+          .set(
```

Comment:
We can use `.cookies`

Reply:
That method does not set the cookies. In the class file, it has this interface
`set(field: "Cookie", val: string[]): this;`